tests/pytorch/r1.13/wav2vec2.libsonnet

```diff
@@ -19,6 +19,7 @@ local utils = import 'templates/utils.libsonnet';
 
 {
   local command_common = |||
+    pip install https://storage.googleapis.com/tpu-pytorch/wheels/torchaudio-nightly-cp37-cp37m-linux_x86_64.whl
```

Comment:
right thing to do might be actually compile and upload a 1.13 wheel, I guess the test might pass as it is.. or you can just pick a date for nightly wheel

Reply:
so 
```
 pip install https://storage.googleapis.com/tpu-pytorch/wheels/torchaudio+yyyymmdd-nightly-cp37-cp37m-linux_x86_64.whl
```

yyyymmdd can be when we cut the 1.13 branch

Reply:
test it locally to see if command actully runs.

Reply:
Not sure how to compile and upload a 1.13 wheel for torchaudio or if there is an instruction. Is it by `python setup.py bdist_wheel` inside pytorch/audio? And how to upload it?

As the alternative, the most recent torchaudio version is tpu-pytorch/wheels/torchaudio-nightly+20220906-cp37-cp37m-linux_x86_64.whl. Is it ok to use it?

Reply:
oh right.. I disabled the audio wheel build a while ago.  using 0906 make sense

Reply:
replaced it.